### PR TITLE
Ntd1hc/feat/vscode extension jsonp

### PR DESCRIPTION
**Pull request by ngoan1608**
Hi Thomas,

Please review the first version of `jsonp` vscode extension and give your feeling about it.

Thank you,
Ngoan

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+-   [Contribution](#contribution)
+-   [Feedback](#feedback)
+-   [About](#about)
+    -   [Maintainers](#maintainers)
+    -   [License](#license)
+
+## Getting Started
+
+This is a vscode extension supports enhanced JSON syntax (is called as `jsonp`)
+which is used by [python-jsonpreprocessor](https://github.com/test-fullautomation/python-jsonpreprocessor) package.
```
Comment (test-fullautomation) [inline README.md]: Hi Ngoan,
syntax is "provided" by python-jsonpreprocessor.
Thank you,
 Thomas

**Review by test-fullautomation — CHANGES_REQUESTED** [review]
Hi Ngoan,
very good!
Please read my finding.
Thank you,
 Thomas

File: README.md (hunk 1 above)
Comment (ngoan1608) [reply README.md]: Hi Thomas,

I have updated it with new commit.

Thank you,
Ngoan

**Review by test-fullautomation — APPROVED** [review]
Hi Ngoan,
looks good to me!
Thank you,
 Thomas

**Pull request merged**